classes/submission/Collector.php

```diff
@@ -56,27 +56,18 @@ public function getQueryBuilder(): Builder
 
         // By issue IDs
         if (is_array($this->issueIds)) {
-            $q->whereIn('s.submission_id', function ($query) {
-                $query->select('issue_p.submission_id')
-                    ->from('publications AS issue_p')
-                    ->join('publication_settings as issue_ps', 'issue_p.publication_id', '=', 'issue_ps.publication_id')
-                    ->where('issue_ps.setting_name', '=', 'issueId')
-                    ->whereIn('issue_ps.setting_value', array_map(strval(...), $this->issueIds));
-            });
+            $q->whereIn('po.issue_id', $this->issueIds);
```

Comment:
This change (and the one to the section filter below) changes the meaning of the query. Previously, it searched for submissions with any publication in the specified issue or section; now, it only fetches results where the *current* publication is in the specified section or issue. I think this change is probably unintentional, right?